internal/rpcserver/rpcserver.go

```diff
@@ -6303,8 +6324,8 @@ func (s *Server) route(ctx context.Context) *http.Server {
 
 	// Websocket endpoint.
 	rpcServeMux.HandleFunc("/ws", func(w http.ResponseWriter, r *http.Request) {
-		authenticated, isAdmin, err := s.checkAuth(r, false)
-		if err != nil {
+		authenticated, isAdmin := s.checkAuth(r, false)
```

Comment:
However, the Websocket endpoint does not.  These changes break that.

Reply:
OK, I will fix this.

Reply:
Reverted.